.github/workflows/build-docker-artifact.yaml

```diff
@@ -1,18 +1,26 @@
 name: "Build tt-metal docker artifact"
 
 on:
-  workflow_call:
   workflow_dispatch:
-
+  push:
+    branches:
+      - dimitri/8082-run-python-tests-on-wheel # Needs to be changed before the merge
+    paths:
+      - dockerfile/ubuntu-20.04-amd64.Dockerfile
```

Comment:
what changes would be needed to do this for 22.04 also?

Reply:
does this support wildcards?
we can do something like -dockerfile/*.Dockerfile or something of that sort

Reply:
Pretty sure it does, fam